bluebutton-data-model-rif/src/main/resources/db/migration/V11__Create_medicare_beneficiary_id_history_table.sql

```diff
@@ -0,0 +1,34 @@
+/*
+ * Creates the MedicareBeneficiaryIdHistory table required by the application. This
+ * schema is derived from the JPA metadata, via the 
+ * gov.hhs.cms.bluebutton.data.pipeline.rif.schema.HibernateSchemaPrinter 
+ * utility.
+ */
+
+${logic.tablespaces-escape} SET default_tablespace = fhirdb_ts2;
+
+create table "MedicareBeneficiaryIdHistory" (
+  "medicareBeneficiaryIdKey" numeric not null,
+  "beneficiaryId" numeric,
```

Comment:
No foreign key here, and I see that we don't have one on `BeneficiariesHistory.beneficiaryId`, either. Any idea why?

Reply:
I created an index on the beneficiaryId later in the file.